src/modules/settings/blockedSkills.js

```diff
@@ -0,0 +1,25 @@
+import querySelector from '../common/querySelector';
+import querySelectorAll from '../common/querySelectorAll';
+import querySelectorArray from '../common/querySelectorArray';
+
+export function checkSkill(skillId) {
+  querySelector(`input[name="blockedSkillList[]"][value="${skillId}"]`)
+    .checked = true;
+}
+
+export function getCheckedSkills() {
+  return querySelectorArray('input[name="blockedSkillList[]"]')
+    .filter((i) => i.checked)
+    .map((i) => i.value)
+    .sort();
+}
+
+export function clearCheckedSkills() {
+  querySelectorAll('input[name="blockedSkillList[]"]')
+    // eslint-disable-next-line no-param-reassign
+    .forEach((i) => { i.checked = false; });
+}
+
+export function submitSkillChanges() {
+  querySelector('input[value="Save Blocked Skill Changes"]').click();
```

Comment:
we should be doing this with AJAX
error = 'Blocked Skill settings updated!'

Reply:
Any idea how to use app to update blocked skills? I'll make the fallback in the meantime.

Reply:
if you can bear with me i'll try to find some time today to look at it

Reply:
i've had a look at this and it looks as if the documentation i have is wrong. hoof is going to send me updated docs on monday. for now just write the da file to only use the fallback. OR just wait until monday because we don't know what the api looks like.

Reply:
ok i've got it working, i'll commit shortly